transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -26,6 +26,9 @@
 
     public static final ChannelOption<Boolean> SO_REUSEPORT = valueOf("SO_REUSEPORT");
 
+    public static final ChannelOption<EpollDomainSocketReadMode> DOMAIN_SOCKET_READ_MODE =
```

Comment:
Yeah, after fixing https://github.com/netty/netty/pull/3345#discussion_r24078705 and https://github.com/netty/netty/pull/3345#discussion_r24078710